### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by dannyroberts**
Indulging in obsession.

Opening for tests.

Last few commits remove code that's no longer used or imported anywhere, as well as https://github.com/dimagi/dimagi-utils/pull/355

### CODE REVIEW

File: corehq/apps/style/forms/widgets.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from django import forms
 from django.forms.fields import MultiValueField, CharField
 from django.forms.utils import flatatt
-from django.forms.widgets import CheckboxInput, HiddenInput, Input, RadioSelect, RadioFieldRenderer, RadioInput, TextInput, MultiWidget
+from django.forms.widgets import CheckboxInput, Input, RadioSelect, RadioFieldRenderer, RadioInput, TextInput, MultiWidget
```
Comment (dimagimon) [inline corehq/apps/style/forms/widgets.py]: E501 line too long (122 > 115 characters)

**Comment by dannyroberts** [comment]
cc: @emord

**Comment by benrudolph** [comment]
lol this is great

**Comment by czue** [comment]
haha nice. how did you do this analysis? we should probably move quickly to avoid all kinds of merge conflicts, but want to make sure you're 100% confident in your process.

**Comment by dannyroberts** [comment]
I started by running `pylint corehq/ --disable=all --enable=unused-import` to get filenames, and then relied on pycharm's own static analyzer and removed the grey imports.

I noticed a couple places where it got it wrong (pycharm can analyze `from ... import *` incorrectly if the module uses `__all__`, thus thinking that an `os` import is overshadowed by the `*` import when it's really not), and those popped up immediately and I fixed them. There's certainly a chance that inside an untested view somewhere there are other cases of this that weren't picked up by my running runserver or running the tests.

I'm not sure how to search for that though.

The reason behind this PR is that I'm working on some simple static analysis of our code to detect functions that are declared but never used, and it's easier for me to do that if we know that if there are no unused imports.

**Comment by benrudolph** [comment]
:bomb:

**Comment by czue** [comment]
guess we're doing it live!

**Comment by benrudolph** [comment]
danny's explanation convinced me :stuck_out_tongue:

**Pull request merged**